targets/posix/nanoCLR/FileStore_POSIX.cpp

```
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+#include <fstream>
+#include <iterator>
+#include <string>
+#include <vector>
+
+// TODO: replace with CLR_RT_FileStore integration once shared runtime code is enabled.
+
+bool nanoCLR_POSIX_LoadFile(const std::string &path, std::vector<unsigned char> &content, std::string &error)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)